### DISCUSSION TITLE
next with express and typescript

### DISCUSSION THREAD

**Discussion by sromexs** [vercel/next.js#17310] [Help] [answered]
hi i want simple next js express js typescript code that back end and front end in the same server
i want next js custom server but what is searched and found in google was old.
is there any new types of next js custom server for new versions ? i relay need that if any one can help answer this thanks.

**Comment by jamesmosier** [comment, marked as answer]
There are a few [examples](https://github.com/vercel/next.js/tree/canary/examples) you could look at that might help:

* [custom-server-typescript](https://github.com/vercel/next.js/tree/canary/examples/custom-server-typescript)
* [custom-server-express](https://github.com/vercel/next.js/tree/canary/examples/custom-server-express)
